format/proto/types.proto

```diff
@@ -14,20 +14,12 @@ enum CheckpointEntryType {
     END = 2;
 }
 
-// The type of requests that can be made to the Orchestrator Service
-enum OrchestratorRequestType {
-    // Query a workflow id
-    QUERY = 0;
-    // Add a new node to the cluster
-    ADD_NODE = 1;
-}
-
 // Orchestrator responses
 enum OrchestratorResponseType {
     // The status of a workflow
     WORKFLOW_STATUS = 0;
     // Id of a created workflow
```

Comment:
this comment does not seem to match variable.

Reply:
why not ? workflow created doesn't necessarily mean add node.

Reply:
I think @medhavidhawan is looking for "Return information about created workflow"